support/nginx/peertube

```diff
@@ -57,6 +57,26 @@ server {
     root /var/www/certbot;
   }
 
+  # Bypass PeerTube for performance reasons. Could be removed
+  # Must be consistent with static client-overrides list in /server/controllers/client.ts
+  location ~ ^/client/(assets/images/(logo\.svg|favicon\.png)) {
+    add_header Cache-Control "public, max-age=31536000, immutable";
+
+    root /var/www/peertube;
+
+    try_files /storage/client-overrides/$1 /peertube-latest/client/dist/$1 $uri;
+  }
+
+  # Bypass PeerTube for performance reasons. Could be removed
+  # Must be consistent with static client-overrides list in /server/controllers/client.ts
+  location ~ ^/client/(assets/images/icons/icon-(36x36|48x48|72x72|96x96|144x144|192x192|512x512)\.png)$ {
```

Comment:
```suggestion
  location ~ ^/client/assets/images/icons/icon-(36x36|48x48|72x72|96x96|144x144|192x192|512x512)\.png$ {
```

Reply:
same here about the parenthesis from `assets`